plotting-service/plotting_service/plotting_api.py

```diff
@@ -367,4 +374,68 @@ async def check_permissions(request: Request, call_next: typing.Callable[..., ty
     raise HTTPException(HTTPStatus.FORBIDDEN, detail="Forbidden")
 
 
+def _latest_image_in_dir(directory: Path) -> Path | None:
+    """Return the newest image file under directory, searching recursively."""
+    latest_path: Path | None = None
+    latest_mtime = float("-inf")
+    for entry in directory.rglob("*"):
+        if entry.is_file() and entry.suffix.lower() in IMAGE_SUFFIXES:
+            mtime = entry.stat().st_mtime
+            if mtime > latest_mtime:
+                latest_path = entry
+                latest_mtime = mtime
+    return latest_path
+
+
+@app.get(
+    "/imat/latest-images",
+    summary="Bundle the latest image from each IMAT sample folder",
+)
+async def get_latest_imat_images() -> FileResponse:
+    """Collect the newest image from each sample folder and serve them as a ZIP bundle."""
```

Comment:
We are returning an image, that is it, not the most recent image from each, just THE most recent image, whatever that is, wherever that is, just THE most recent image.